test/bot_test.js

```diff
@@ -11,13 +12,15 @@ const USER_FROM_REPOSITORY = { token: TOKEN, username: ANY_USERNAME }
 
 describe('Bot', function() {
   it('returns message ok on track today command', function() {
-    var userRepositoryMock = sinon.mock(userRepository)
+    const userRepositoryMock = sinon.mock(userRepository)
     userRepositoryMock.expects("findFromUsername").once().withArgs(ANY_USERNAME).returns(USER_FROM_REPOSITORY)
     request = requestBuilder().withText("today").withUserName(ANY_USERNAME)
 
     response = bot(request)
 
+    userRepositoryMock.verify()
     assert.equal('Ciao ' + ANY_USERNAME + '. Ho tracciato la giornata di oggi.', response)
+    userRepositoryMock.restore()
```

Comment:
`restore` the original object methods, and in this case it's especially important because in `user_repository.js` we export an instance (it's actually a singleton i think) with `module.exports = new UserRepository(db)`